consensus/spos/bn/bnSubroundsFactory.go

```diff
@@ -13,6 +15,8 @@ type factory struct {
 	consensusCore  spos.ConsensusCoreHandler
 	consensusState *spos.ConsensusState
 	worker         spos.WorkerHandler
+
+	appStatusHandler core.AppStatusHandler
```

Comment:
unused variable?

Reply:
variable is used now, because i forgoted to increment MetricCountAcceptedBlocks in bn/subroundEndRound.go

Reply:
increment is done on bnEndSubround. This is still unused

Reply:
now its used